test/development/app-dir/dynamic-io-dev-errors/dynamic-io-dev-errors.test.ts

```diff
@@ -64,11 +65,14 @@ describe('Dynamic IO Dev Errors', () => {
     )
 
     const description = await getRedboxDescription(browser)
-    const stack = await getRedboxCallStack(browser)
 
     expect(description).toMatchInlineSnapshot(
       `"[ Server ] Error: Route "/no-accessed-data": A component accessed data, headers, params, searchParams, or a short-lived cache without a Suspense boundary nor a "use cache" above it. We don't have the exact line number added to error messages yet but you can see which component in the stack below. See more info: https://nextjs.org/docs/messages/next-prerender-missing-suspense"`
     )
+
+    // Expand the stack frames, since the first frame `Page [Server] <anonymous>` is treated as ignored.
```

Comment:
We don't usually ignore `anonymous` anywhere else. Why do we ignore it in the Redbox? Ignoring is tricky since all component stack frames of RSCs are anonymous. Right now we error on the side of showing too much. Let's opt for consistency for now to not break user expectations and leave this discussion for later if it turns out we're still showing too much on average.

Reply:
we have a built-in logic to ignroe the `anonymous` before where we added in early 14, but it's actually not super helpful since sometimes it's insightful. I think we'll remove that filter later.

checked with Seb that we also want to keep this line. I'll add a TODO